### PR TITLE
Feature/scio 081

### PR DESCRIPTION
**Pull request by peel**

### CODE REVIEW

File: src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/Enrich.scala [hunk 1]
```diff
@@ -88,7 +88,7 @@ object Enrich {
         System.exit(1)
       case Right(config) =>
         run(sc, config)
-        sc.close()
+        sc.run()
```
Comment (lukeindykiewicz) [inline src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/Enrich.scala:91]: `close` changed to `run`?

**Review by lukeindykiewicz — APPROVED** [review]
lgtm

File: src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/Enrich.scala (hunk 1 above)
Comment (peel) [reply src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/Enrich.scala:91]: Indeed: https://github.com/spotify/scio/pull/2083/files

Comment (lukeindykiewicz) [reply src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/Enrich.scala:91]: thanks

Comment (dilyand) [reply src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/Enrich.scala:91]: In `0.7.0` the return type used to be this:

```
def close(): ScioResult
```

Now, in `0.8.0` it's been changed to:

```
def close(): ScioExecutionContext
```

The new `run()` also returns a `ScioExecutionContext`. But if we want to still return a `ScioResult`, I think we need to also call:

```
def waitUntilDone(
    duration: Duration = awaitDuration,
    cancelJob: Boolean = cancelJob
  ): ScioResult
```

This is how I've done it in other DF jobs:

```
import scala.concurrent.duration.Duration

...
sc.run().waitUntilDone(Duration.Inf)
```

Comment (peel) [reply src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/Enrich.scala:91]: Here we return unit and ignore the result altogether. I'd leave just `run()` unless we want this value to be used in some other way.

Comment (dilyand) [reply src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/Enrich.scala:91]: Ah, I missed the additional `()`.
👍🏻

**Review by dilyand — APPROVED** [review]
LGTM. 👍🏻

**Pull request merged**